### DISCUSSION TITLE
Opt-out of auto updates

### DISCUSSION THREAD

**Discussion by caspiano** [mozilla/hubs#2820] [General Questions & Discussion] [answered]
It would be ideal if there was an option to opt-out of auto-updates.
These changes can happen and have happened at the worst of times.

**Comment by robinkwilson** [comment]
Sorry to hear that! I'm assuming you're running your own Hubs Cloud instance. 

**Can you make an issue in https://github.com/mozilla/hubs-cloud/issues so you can track your request?**

Link it here, so other people can vote for it (with a thumbs up).

**Comment by caspiano** [comment, marked as answer]
Issueified 😺  mozilla/hubs-cloud/issues/96

**Reply by robinkwilson** [reply]
Anyone that wants to opt out of auto updates, to **vote** for this to be considered a new feature: 

1. **Thumbs up** this discussion 
2. Go to and **thumbs up** https://github.com/mozilla/hubs-cloud/issues/96

**Comment by gfodor** [comment]
Reminder that this feature is hard to build since new instances will boot
up to AMI version of the code. To do this properly, the system will need to
start tracking what version the user has deployed in external storage and
use that on boot to bootstrap somehow.

**Comment by robinkwilson** [comment]
@netpro2k mentioned in discord

We try to be careful to keep the backend compatible with old client changes. The last time we made a breaking backend change was when we switched from Janus to MediaSoup and we gave over a month of notice.
If something else has broken, that was a bug and we would like to know about it.
That said, yeah it should be possible to disable backend updates, though its not really supported. High level you need to get your habitat supervisor to stop monitoring the polycosm-stable-a channel (and monitor nothing)

You can do this one off by sshing into the servers one by one and restarting the services without change monitoring enabled, but it will not apply to new instances. To do that you would need to modify the boot script, or just add commands to the end of the bootup sequence in the cloudformation template.
